### ISSUE TITLE
Build failing due to quay.io permissions

### ISSUE THREAD

**Issue by jprice-da15252** [mongodb/terraform-aws-ecs-task-definition#18]
CircleCI is failing at pre-commit stage due to the following:
```
Starting container quay.io/mongodb-build/terratest
  image cache not found on this host, downloading quay.io/mongodb-build/terratest

Error response from daemon: unauthorized: access to the requested resource is not authorized
```

**Comment by jasonwalsh** [comment]
Hi @jprice-da15252 thanks for bringing this to my attention. I will fix this.

**Comment by jasonwalsh** [comment]
Fixed via f1773d1
